src/initializers/log4js/json-appender.ts

```diff
@@ -0,0 +1,42 @@
+import * as lodash from 'lodash';
+
+const jsonAppender = ({ filter_status = [] }) => {
+  return logEvent => {
+    let event = logEvent.level.levelStr === 'ERROR' ? createErrorLog(logEvent) : createValidLog(logEvent);
+    var json = JSON.stringify(event);
+    console.log(json);
```

Comment:
What does log4js use internally? Maybe process.stdout.write is more suitable

Reply:
Done (kept console as it has only memory overhead)

Reply:
👍